.golangci.yaml

```diff
@@ -46,15 +46,6 @@ linters-settings:
     ignored-functions:
       - strings.SplitN
 
-  govet:
```

Comment:
Why were these removed?

Reply:
Because the Golang ljnt keep showing "Warning" and it requires them to be replaced with Shadow configuration.

Reply:
I think we should leave the linter config file as is, and just fix the problems the linter is raising